maven-model-builder/src/main/java/org/apache/maven/model/validation/DefaultModelValidator.java

```diff
@@ -89,6 +85,15 @@
 
     private final Set<String> validProfileIds = new HashSet<>();
 
+    @Inject
+    private ModelVersionProcessor versionProcessor;
```

Comment:
Any reason why this is not on ctor injection?

Reply:
Changed to ctr injection